Core.cpp

```diff
@@ -45,10 +48,18 @@ int main(int, char**)
 
 		if (bboxes.size() && !flag) {
 			//High Beam detected
+
+			detected_image = frame; //save detected frame;
+
+			sprintf(file,"Image_%d.jpg",count);  //save files with different names using 'count'     
```

Comment:
`file` isn't defined anywhere I think. Also, C++ has direct string concatenation with `+` operator. Can probably write this as `string file = "Image_"+to_string(count)+".jpg";`